sources/__tests__/accessible-name.js

```diff
@@ -283,7 +294,7 @@ describe("slots", () => {
 test.each([
 	[
 		`
-<div data-test aria-labelledby="label">I reference my name</div>
+<element1 data-test aria-labelledby="label">I reference my name</element1>
```

Comment:
Let's update the expectation here instead. I'm not so sure about `<element1>` since that's invalid HTML I suppose?